source/tests/unit/xnet_converters_tests.cpp

```diff
@@ -1,830 +1,116 @@
 #define NOMINMAX
 
-#include <custom/xnetsocket_converters.h>
+#include <custom/nixnet_converters.h>
 #include <gmock/gmock.h>
 #include <gtest/gtest.h>
-#include <nixnetsocket/nixnetsocket_mock_library.h>
+#include <nixnet/nixnet_mock_library.h>
 
 #include <algorithm>
 #include <numeric>
 #include <string>
 
-using namespace nixnetsocket_grpc;
+using namespace nixnet_grpc;
 using namespace ::testing;
 namespace pb = ::google::protobuf;
 using namespace std::string_literals;
-using ni::tests::unit::NiXnetSocketMockLibrary;
-using nidevice_grpc::converters::allocate_output_storage;
-using nidevice_grpc::converters::convert_from_grpc;
-using nidevice_grpc::converters::convert_to_grpc;
+using ni::tests::unit::NiXnetMockLibrary;
+
+namespace nixnet_grpc {
```

Comment:
These are methods implemented in `nixnet_service.custom.cpp`, right? It would be better to put these in a header (`nixnet_service.custom.h`?) and also a separate sub-namespace (maybe `internal` or `converters` or `helpers`). The implementation could also be moved to a separate cpp file (i.e., `nixnet_converters.cpp` or `nixnet_service_custom_helpers.cpp`) if you're concerned about adding the header "just for tests".

Or ... at least put a comment, because this is not common. Or you can use some other compromise solution if you prefer. I realize we're not very strict about cpp file organization in grpc-device, and adding these tests is more important than getting it perfect!

Reply:
Created a seperate namespace as converters in nixnet_service_custom.cpp and and nixnet_converters.h header file